manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon

```

Comment:
```suggestion
PackageIdentifier: K-Meleon.K-Meleon
```

Reply:
You'll also need to update the folder names and the `.yaml` files using GitHub's desktop client.